radiotools/atmosphere/models.py

```diff
@@ -705,8 +705,8 @@ def _get_density(self, zenith, xmax):
 #         dxmax_geo = self._get_distance_xmax_geometric(xmax, observation_level=0)
 #         return self._get_density_for_distance(dxmax_geo)
 
-    def _get_density_for_distance(self, d, zenith):
-        h = get_height_above_ground(d, zenith)
+    def _get_density_for_distance(self, d, zenith, observation_level=0):
+        h = get_height_above_ground(d, zenith, observation_level)
```

Comment:
do I see it correctly that this change does not have any effect? `_get_density_for_distance` is an external function that is never called with a third argument, and the default value for `observation_level` in `observation_level` is anyway zero. But adding the argument explicit doesn't harm.

Reply:
Hi,
yes exactly. Within the existing code this change changes nothing. If there is never the need to add an observation level to this calculation we might want no to add it (there is the function `get_density_for_distance` which has this argument anyway. 
Are you happy with the function I added?